### ISSUE TITLE
The generated file doesn't bypass Windows Defender.

### ISSUE THREAD

**Issue by ghost** [UnamSanctam/UnamDownloader#49]
The built file doesn't bypass Windows Defender, so there is no point in using it.

![imagen_2022-11-01_185612881](https://user-images.githubusercontent.com/97046121/199304173-06315ad2-f2bc-49ea-a374-1a9108293243.png)
![imagen_2022-11-01_185644194](https://user-images.githubusercontent.com/97046121/199304275-f6f8f254-7555-4350-8d06-ab95ad1489f9.png)

**Comment by UnamSanctam** [comment]
Obviously, the last release was May 16, programs are usually only undetected for about a week after release, though obfuscators and crypters can still be used to reduce detections. The UnamDownloader and UnamBinder are going to get updates now that the miner has been updated (released it yesterday) since they will use code from the miner.

**Comment by ghost** [comment]
Alr thanks, im new in all of this av evasion thing.

(bdw thanks for the fast answer <3)
